### ISSUE TITLE
[Tokenizer]: “N-more” overflow popover is not openable via keyboard

### ISSUE THREAD

**Issue by akrougliy** [UI5/webcomponents#12882]
### Bug Description

The “N-more” overflow popover in the tokenizer should be openable via keyboard, as per the design (accessibility) specification, but this currently does not seem to work.

### Affected Component

Tokenizer

### Expected Behaviour

The overflow popover should be accessible via keyboard as defined in the design (accessibility) specification.

### Steps to Reproduce

1.
2.
3.
...

### Priority

Low

### UI5 Web Components Version

 2.19.0-rc.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by StefanDimitrov04** [comment]
Hello @UI5/ui5-team-ctr-ril, 

As this behaviour requires an enhancement, after discussion with the team, the issue will be handled as a feature request, and a corresponding backlog item will be created.

Regards,
Stefan

**Comment by akrougliy** [comment]
Evaluation of the behavior is required, hence closing this item.
